src/main/scala/com/databricks/spark/xml/util/InferSchema.scala

```diff
@@ -101,7 +101,7 @@ private[xml] object InferSchema {
 
           Some(inferObject(parser, options, rootAttributes))
         } catch {
-          case NonFatal(_) if shouldHandleCorruptRecord =>
+          case NonFatal(x) if shouldHandleCorruptRecord =>
```

Comment:
I think `_` is fine since that's not used here.

Reply:
I will check it